heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
+    "label": "HeapOverflow",
+    "display": "standalone",
+    "access": "external",
+    "image": {
+        "name": "heapoverflow",
+        "tags": [
+            "latest"
+        ]
+    },
+    "logo": "",
+    "info": "",
+    "depends": [],
+    "config": [],
+    "ports": [],
+    "volumeMounts": [],
+    "additionalResources": [],
+    "repositories": [],
+    "command": [],
+    "args": [],
+    "catalog": "user",
```

Once this PR is merged, this will now be a `system` spec, meaning it comes from the official catalog :+1:
```suggestion
    "catalog": "system",
```